src/Symfony/Component/Messenger/Bridge/AmazonSqs/Transport/AmazonSqsQueryStringParser.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Bridge\AmazonSqs\Transport;
+
+/**
+ * @author Yukihiro Arisawa <goreboothero@gmail.com>
+ */
+class AmazonSqsQueryStringParser
```

Comment:
I don't think it is worth to create a new class for parsing the URL. It is more simple to inline it to the `Connection` class like before.

Reply:
@YaFou 
I thought it would be easier to test if it was divided into classes, but let's go with the reviewer's opinion here.

I have removed this class.👍